source/Applications/Advanced/MaskPointCloud/MaskPointCloud.cpp

```diff
@@ -11,20 +11,24 @@ The ZDF file for this sample can be found under the main instructions for Zivid
 #include <pcl/visualization/cloud_viewer.h>
```

Comment:
Update to `#include <pcl/visualization/pcl_visualizer.h>` if you decide to use that visualizer